lib/hawkular/inventory/inventory_api.rb

```diff
@@ -485,7 +485,8 @@ def events(type = 'resource', action = 'created')
       }
       ws_options[:headers][:'Hawkular-Tenant'] = tenant_id
 
-      url = "#{entrypoint.gsub(/https?/, 'ws')}/ws/events?tenantId=#{tenant_id}&type=#{type}&action=#{action}"
+      url = "#{entrypoint.sub(/https/, 'wss').sub(/http/, 'ws')}/ws/events?"\
+      "tenantId=#{tenant_id}&type=#{type}&action=#{action}"
```

Comment:
[Made a separate method](https://github.com/hawkular/hawkular-client-ruby/pull/129) to handle the scheme manipulation, since I'm sure i'll use it elsewhere and I don't want to be using sub directly.